### DISCUSSION TITLE
WildStacker , Spawners

### DISCUSSION THREAD

**Discussion by BlazzeKiller** [OmerBenGera/WildStacker#962] [Approved Feature Requests] [answered]
WildStacker , Can Spawners be changed to spawn Anywhere mobs, like for example Cows , can i change so that cows instead of grass block, drop from air spawned just like Iron Golems?

**Comment by BlazzeKiller** [comment]
Cannot Spawn Entities due to on Grass restriction.

**Comment by OmerBenGera** [comment, marked as answer]
Yes, it is possible.
Look for the spawn-conditions section in the config file.
